src/App.js

```diff
@@ -7,50 +7,52 @@ class App extends Component {
   constructor (props) {
     super(props)
     this.state = {
-      // products: ['stuff', 'things'],
+      // products: [{'name': 'stuff', 'url': 'example.com', 'price': '$50'}, {'name': 'things', 'url': ''}],
```

Comment:
There's no reason to leave commented code in a pull-request. That's what source-control is for.